app/src/main/java/com/orgzly/android/usecase/LinkFindTarget.kt

```diff
@@ -15,9 +17,23 @@ class LinkFindTarget(val path: String) : UseCase() {
     }
 
     private fun openLink(dataRepository: DataRepository, path: String): Any {
+        val regex = """(.*)\:\:(.*)""".toRegex()
```

Comment:
Perhaps `*` can be included in the regex. Something like:

```suggestion
        val regex = """(.*)::\*(.*)""".toRegex()
```
If not, we need to check for it below, having in mind support for other link types in the future, like:

```
file:sometextfile::NNN (jump to line number)
file:projects.org::some words (text search)
file:projects.org::#custom-id (headline search)
```